examples/eks/prometheus-pipeline/eks-prometheus-sidecar.yaml

```diff
@@ -71,8 +71,10 @@ data:
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: adotcol-admin
+  name: amp-iamproxy-ingest-service-account
   namespace: adot-col
+  annotations:
+    eks.amazonaws.com/role-arn: arn:aws:iam::YOUR_ACCOUNT_ID:role/amp-iamproxy-ingest-role
```

Comment:
Can we use `<>` for the variables in the template?

Reply:
I'll change that throughout our template